src/missions/uninitialized-stack-frame-control-flow/README.md

```diff
@@ -1,3 +1,44 @@
 # Exploiting an uninitialized stack frame to manipulate control flow
 
-*This section remains under implementation.*
+The objective of this mission is to demonstrate arbitrary code
+execution through the use of uninitialized variables on the stack,
+despite CHERI protections.  You will attack three different versions
+of the program:
+
+1. A baseline RISC-V compilation, to establish that the vulnerability is
+   exploitable without any CHERI protections.
+
+2. A hardened CHERI-RISC-V compilation with stack clearing, which
+   should be non-exploitable.
+
+3. A baseline CHERI-RISC-V compilation with no stack clearing, which
+   should be non-exploitable due to pointer tagging.
```

Comment:
It's exploitable when the input matches '^-*$`, no?

Reply:
I should perhaps rename `nothing_doing` to `no_cookies` to be more inline with the theme.